Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:6
```

Comment:
If there aren't further problems I don't foresee any problem in having a dedicated folder. Sounds nice to me.